scripts/usb_scan.py

```diff
@@ -616,7 +616,11 @@ def to_pusb(device):
         log.logToSyslog(level=logging.WARNING)
 
     # get usb info
-    devices, interfaces = get_usb_info()
+    try:
+        devices, interfaces = get_usb_info()
+    except Exception as e:
+        log.error("Failed to get usb info: {}".format(str(e)))
```

Comment:
Make `log_exit` global and here use `log_exit` as well

Reply:
Done.